config/locales/de.yml

```diff
@@ -322,6 +322,54 @@ de:
       title: 'Titel'
       teacher: 'DozentIn'
       editors: 'EditorInnen'
+    annotation:
+      annotations: 'Annotationen'
+      annotation_modal_head: 'Annotation anlegen'
+      inherit_from_lecture: 'von Vorlesung erben'
+      time: 'Zeit:'
+      comment: 'Kommentar'
+      color: 'Farbe'
+      category: 'Kategorie'
+      note: 'Notiz'
+      content: 'Inhalt'
+      mistake: 'Fehler'
+      presentation: 'Darstellung'
+      where_problem: >
+        Wo liegt das Problem?
+      definition: 'Definition'
+      argument: 'Argument'
+      strategy: 'Beweisstrategie'
+      previous_annotation: 'vorherige Annotation'
+      annotation_position: 'zur Annotation springen'
+      edit_annotation: 'Annotation editieren'
+      close_annotation_area: 'Feld schließen'
+      next_annotation: 'nächste Annotation'
+      further_help: 'Du brauchst weitere Hilfe? Dann klicke auf:'
+      warning_message:
+        publishing: >
+          Jeder, der diese Vorlesung abonniert hat, wird deinen Kommentar
+          lesen können, sobald du auf "Speichern" klickst.
+          Falls du das nicht möchtest, entferne bitte wieder den Haken
+          bei "Als Kommentar veröffentlichen?".
+        mistake: >
+          Bevor du diesen Kommentar abschickst, überprüfe außerdem bitte,
```

Comment:
We have consistently capitalized `du` so far. Note this is not the only line where you've used `du` instead of `Du`. Watch out for simple search and replace, there's lots of variants like `Deinen`, `Dich` etc.

Reply:
Good catch! Thanks to your comment, I also found a type in multiple_close_annotations_2 ;).